feature/floor-map/src/main/java/io/github/droidkaigi/confsched2023/floormap/FloorMapScreen.kt

```diff
@@ -115,42 +126,106 @@ private fun FloorMapScreen(
             )
         },
         content = { innerPadding ->
-            Box(
+            BoxWithConstraints(
                 Modifier
                     .padding(innerPadding)
                     .padding(horizontal = 16.dp),
             ) {
-                Column(
-                    Modifier
-                        .fillMaxSize(),
-                ) {
-                    FloorMap(uiState = uiState.floorMapUiState)
-                    FloorMapSideEventList(
-                        uiState = uiState.floorMapSideEventListUiState,
+                if (widthSizeClass != WindowWidthSizeClass.Compact) {
```

Comment:
If large device, use large layout.

Reply:
In FloorMapContent(), we will have the `if (widthSizeClass != WindowWidthSizeClass.Compact) `

Reply:
>  how about using the Slot API pattern here to simplify the layout? 👀

Thank you. It certainly looks better that way, so I will give it a try.

But, LargeScreen needs 2 FloorMaps, MobileScreenContent needs 1 FloorMaps. So maybe will be more complicated.
I'll look for a simpler way to write based on the advice you gave me.

```
LargeScreenContent
  ┗ Row
    ┗ Column
      ┗ FloorMap(uiState = FloorMapUiState.of(Basement))
      ┗ FloorMapSideEventList
    ┗ Column
      ┗ FloorMap(uiState = FloorMapUiState.of(Ground))
      ┗ FloorMapSideEventList

MobileScreenContent
  ┗ Column
    ┗ FloorMap(uiState = uiState.floorMapUiState)
    ┗ FloorMapSideEventList
    ┗ FloorLevelSwitcher
```

Reply:
> In FloorMapContent(), we will have the if (widthSizeClass != WindowWidthSizeClass.Compact) 

Sorry, I overlooked this one.

Reply:
fixed https://github.com/DroidKaigi/conference-app-2023/pull/814/commits/b65e11589113c5025d5839cb51d7529d825c537b